ingest-app/src/cmr/ingest/validation/validation.clj

```diff
@@ -100,7 +100,10 @@
                    (if (mt/umm-json? (:format concept))
                      (let [umm-version (mt/version-of (:format concept))
                            accept-version (config/ingest-accept-umm-version (:concept-type concept))]
-                       (if (>= 0 (compare umm-version accept-version))
+                       ;; when the umm-version goes to v1.10 and accept-version is v1.9, we need
+                       ;; to compare the numbers after the dot.  
+                       (if (>= 0 (compare (Integer. (last (str/split umm-version #"\.")))
```

Comment:
This will not work for comparing `1.9` and `2.1`. We should convert the versions to float first, then compare them.

Reply:
I thought about that, but we can't guarantee that it'll be a float number either.  what if we will have 1.10.1 or sth like that?

Reply:
That's a good point. We should split the number by `.` and then compare the integer in orders.